Dockerfile

```diff
@@ -70,6 +70,11 @@ RUN git clone https://github.com/openmm/openmm.git --branch "${OPENMM_VERSION}"
     make -j "$(nproc)" install && \
     make PythonInstall
 
+# Install pre-commit and cache hooks
+RUN pip install --no-cache-dir pre-commit
```

Comment:
This potentially can hide cases where pre-commit releases a new version that no longer works with our configuration. If we want to pin this here as well in the setup.py that would be a potential solution?

Reply:
Pinned pre-commit in both places in c82f820